components/gitpod-db/src/typeorm/migration/1628245953986-UpdateExamples.ts

```diff
@@ -0,0 +1,48 @@
+/**
+ * Copyright (c) 2021 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License-AGPL.txt in the project root for license information.
+ */
+
+ import {MigrationInterface, QueryRunner} from "typeorm";
+
+ export class UpdateExamples1628245953986 implements MigrationInterface {
+
+ public async up(queryRunner: QueryRunner): Promise<any> {
+         let priority = 90;
+         const newEntries = [
+             { url: 'https://github.com/gitpod-io/template-typescript-node', description: 'A Node.js app, written in TypeScript.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-typescript-react', description: 'A create-react-app template, written in TypeScript.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-python-django', description: 'A Django app template. ', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-python-flask', description: 'A Flasker app template.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/spring-petclinic', description: 'A Spring app written in Java.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-php-drupal-ddev', description: 'A Drupal app template, scaffolded by DDEV.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-php-laravel-mysql', description: 'A Laravel app template, with MySQL.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-ruby-on-rails', description: 'A Ruby on Rails app template, with Postgres. ', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-golang-cli', description: 'A CLI template, written in Go.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-rust-cli', description: 'A CLI template, written in Rust.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-dotnet-core-cli-csharp', description: 'A CLI starter for .NET written in C#.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-sveltejs', description: 'A Svelte.js app writtten in JavaScript.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-sveltejskit', description: 'A SvelteKit app template.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-datasette', description: 'A Datasette template.', priority: priority-- },
+             { url: 'https://github.com/gitpod-io/template-nix', description: 'A nix template for reproducible development environments.', priority: priority-- },
```

Comment:
isn't reproducible development environments what Gitpod is about?

Reply:
Nix is a _really_ is reproducible - https://r13y.com/ whereas Docker pretends to be (`RUN sudo apt-get upgrade` == fail). What verbiage should be use instead?

<img width="1202" alt="2021-08-06_21-10-48" src="https://user-images.githubusercontent.com/127353/128501985-aacdaa9b-5c42-4ade-9ad3-cb3d4d7e6459.png">

Reply:
proposal: `A template for reproducible development environments via nix.`